src/clients/storage/GraphStorageClient.cpp

```diff
@@ -623,6 +623,82 @@ StorageRpcRespFuture<cpp2::ScanVertexResponse> GraphStorageClient::scanVertex(
       });
 }
 
+folly::SemiFuture<StorageRpcResponse<cpp2::KVGetResponse>> GraphStorageClient::get(
```

Comment:
Could the `GraphStorageClient` be renamed to `StorageClient`?

Reply:
👌